README.md

```diff
@@ -213,63 +223,69 @@ You may wish to read this Guide all the way through, front to back, or jump to a
 area, depending on your familiarity with the subject.
 
 Equity compensation lies at the intersection of corporate law, taxation, and employee
-compensation, and so requires some basic understanding of all three topics.
+compensation, and so requires some basic understanding of all three.
 You might think compensation and taxation are separate topics, but they are so intertwined
-that it’s hard to explain one without the other.
+it would be misleading to explain one without the other.
 
-So we’ve laid out key elements of these subjects, from the ground up:
+So we’ve laid out the key elements of these subjects, from the ground up:
 
-- [**Fundamentals of Stock Corporations**](#fundamentals-of-stock-corporations): What is equity?
-  To understand the basics, you need to know how companies organize their ownership and the
-  basics of how stock works.
-- [**Equity Compensation Basics**](#equity-compensation-basics): What is equity worth?
-  The next set of fundamentals are around compensation and if and when you can sell equity
-  for cash.
-- [**How Equity is Granted**](#how-equity-is-granted): Equity is granted in many forms, most
-  commonly restricted stock grants, stock options, and RSUs.
-  This covers exactly what these are.
+- [**Fundamentals of Stock Corporations**](#fundamentals-of-stock-corporations): To understand
+  exactly what equity is, it’s essential to know how stock works and how companies at
+  different stages organize their ownership.
+- [**Equity Compensation Basics**](#equity-compensation-basics): Here we dive into the fundamentals
+  of compensation and when and whether you can sell equity for cash.
+  These are important concepts to learn in order to understand what equity might be worth.
+- [**How Equity is Granted**](#how-equity-is-granted): This section covers the forms in which equity
+  is most commonly granted, including restricted stock grants, stock options, and RSUs.
 
 Next is taxes:
 
 - [**Tax Basics**](#how-equity-is-granted): Many of the headaches of equity compensation involve how
   it is taxed. We go back to basics again, this time around taxation, for anyone who needs a
-  refresher on the kinds of taxes and tax rates in the United States.
-- [**Taxes on Equity Compensation**](#taxes-on-equity-compensation): What kind of equity you have,
-  the state of your company, and your decisions around when you exercise and pay tax can
-  massively affect how much tax you owe tax.
+  refresher on tax rates and types of taxes in the United States.
+- [**Taxes on Equity Compensation**](#taxes-on-equity-compensation): How much tax you owe is
```

Comment:
Revert this item or rewrite carefully.

Reading the new version I think we're implying we are only talking about stock options, which isn't the case, and omitting the choice of when you pay tax (which is 83(b)s).